modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -223,13 +238,26 @@ public function getItems(AlephMaterial $material) {
    * @param \Drupal\aleph\Aleph\Entity\AlephPatron $patron
    *    The patron to get loans from.
    *
-   * @return \SimpleXMLElement
+   * @param $loan_id
+   *    The loan ID to get specific loan.
+   *
+   * @return \SimpleXMLElement The response from Aleph.
    *    The response from Aleph.
    *
-   * * @throws \RuntimeException
+   * @throws \RuntimeException
    */
-  public function getLoans(AlephPatron $patron) {
-    return $this->requestRest('GET', 'patron/' . $patron->getId() . '/circulationActions/loans?view=full');
+  public function getLoans(AlephPatron $patron, $loan_id = FALSE) {
```

Comment:
Not 100% specific to this line - but in general we should be carefull to differentiate between safe data that comes from objects we've created (and objects in general) - and potentially dangerous "raw" data that might originate directly from a url-parameter that the user can modify. (think injection attacks)